### ISSUE TITLE
config.plist issues 

### ISSUE THREAD

**Issue by KelsonTimaru** [Lost-Entrepreneur439/thinkpad-t480s-hackintosh#8]
1. I had to set **HideAuxiliary**=**false** to bring up "install MacOS" menu, otherwise only "Boot Windows" available there. 
2. I had to set Misc Security **SecureBootModel** = **disabled** as it keep rebooting at the 2nd phase

**Comment by EliasGabr** [comment]
yes and when you finish installing you can also re enable applesecureboot and install airportitlwm, then you can use native macos wifi instead of connecting through heliport

**Comment by KelsonTimaru** [comment]
Thank you EliasGabr for your reply, I will set SecureBootModel back then.
I've got HeliPort installed as I did not know AirPorititlwm, and I don't know how to install AirPorititlwm.

A quick research hope it's what I need:

https://www.reddit.com/r/hackintosh/comments/oyufvq/how_to_use_airportitlwm_based_on_my_own/

**Comment by KelsonTimaru** [comment]
Thanks, I didn't find airportitlwm and I was going to download and install it manually. 
Then I realised you must be talking about Ventura while I installed Sonoma, I guess that's why. Phew....

So I guess airportitlwm does not work on Sonoma yet?

**Comment by EliasGabr** [comment]
it works, here is the link for airportitlwm https://github.com/OpenIntelWireless/itlwm/releases/tag/v2.3.0

here is a step by step guide again on how to install airportitlwm

1. Download Sonoma 14.4 kext
2. put the airportitlwm.kext into kext folder in efi partition
3. remove itlwm.kext
4. open efi partition's config.plist using propertree
5. do the oc snapshot using Ctrl-r or win-r on mac then select OC folder of EFI partition then select open on bottom right
6. then change securebootmodel to default
7. save changes to File and restart

**Comment by KelsonTimaru** [comment]
Thank you sooo much!
Not quite understand about 5 the snapshot, however I'll give it a try. 

You have a great day my friend!

**Comment by KelsonTimaru** [comment]
> it works, here is the link for airportitlwm https://github.com/OpenIntelWireless/itlwm/releases/tag/v2.3.0
> 
> here is a step by step guide again on how to install airportitlwm
> 
> 1. Download Sonoma 14.4 kext
> 2. put the airportitlwm.kext into kext folder in efi partition
> 3. remove itlwm.kext
> 4. open efi partition's config.plist using propertree
> 5. do the oc snapshot using Ctrl-r or win-r on mac then select OC folder of EFI partition then select open on bottom right
> 6. then change securebootmodel to default
> 7. save changes to File and restart

Thanks my friend, airportitlwm works great so far!!!
